syntax = "proto3";

option go_package = "/main";

package helloworld;

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
  bytes vHiveMetadata = 15;
}

message HelloReply {
  string message = 1;
}